runtime/src/common/sgx/ias.rs

```diff
@@ -309,6 +327,14 @@ pub fn verify(avr: &AVR, policy: &QuotePolicy) -> Result<VerifiedQuote> {
         return Err(AVRError::ProductionEnclave.into());
     }
 
+    // If the minimum TCB evaluation data number is present in the policy,
+    // then the report's shouldn't be too low.
+    if policy.min_tcb_evaluation_data_number > 0
+        && avr_body.tcb_evaluation_data_number()? < policy.min_tcb_evaluation_data_number
```

Comment:
This could probably be simplified by having the `avr_body.tcb_evaluation_data_number` return `0` in case the number is missing. And then the check can just be whether the number is lower than the minimum (in case the policy specifies one).